easybuild/easyconfigs/d/DIAMOND/DIAMOND-2.0.6-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'DIAMOND'
+version = '2.0.6'
+
+github_account = 'bbuchfink'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
```

Comment:
Please use an explicit URL without template strings for the `homepage`. We do not have any tests for it, but we try to avoid obfuscated homepages, otherwise it is complicated to load the link from the easyconfig.

Reply:
@lexming I changed this.